src/MICore/LaunchOptions.xsd.types.designer.cs

```diff
@@ -13,8 +13,9 @@
 // 
 namespace MICore.Xml.LaunchOptions {
```

Comment:
>namespace [](http://example.com/codeflow?start=0&length=9)

You either need to reset this file back to where it was (recommended approach) or make the changes to the LaunchOptions XSD and regenerate it.

Reply:
yep you can run
`git checkout origin/main src/MICore/LaunchOptions.xsd.types.designer.cs` to reset it back to the state in `main`